Common/MapGenerator.ts

```diff
@@ -507,6 +507,7 @@ export class MapGenerator
                         suboption.IsList = this.Type_IsList(p.modelType);
                         suboption.TypeName = this.Type_Name(submodel);
                         suboption.TypeNameGo = this.TrimPackageName(suboption.TypeName, this.Namespace.split('.').pop());
+                        suboption.TypeNameGo = suboption.TypeNameGo.charAt(0).toUpperCase() + suboption.TypeNameGo.slice(1);
```

Comment:
I think you could just use common function here:
https://github.com/Azure/autorest.cli/blob/2dabaee25b8db22ff5bff7725d3e0f8e18caaa25/Common/Helpers.ts#L19